pkgs/stdenv/generic/setup.sh

```diff
@@ -385,6 +385,25 @@ appendToVar() {
     fi
 }
 
+removeAllPrefixedFromVar() {
```

Comment:
Yes, this is something that would be useful globally, but should probably stay in `module.nix` until the testing is done.